docs/examples/hypervisors/kvm/kvm-baremetal/datasource.tf

```diff
@@ -0,0 +1,17 @@
+data "oci_identity_availability_domains" "ads" {
+  compartment_id = "${var.tenancy_ocid}"
+}
+
+data "oci_core_images" "base-image" {
+  compartment_id           = "${var.compartment_ocid}"
+  operating_system         = "Oracle Linux"
+  operating_system_version = "7.3"
```

Comment:
This will no longer work, see #342 and #345. Recommend to filter on a specific display name instead, see #346.

Reply:
It's looks like OL 7.4 broken the code. I will have to take a look on that.

Reply:
Never mind. The script is not broken, but I need some controls to avoid using the wrong compute shape on both baremetal and nested scenarios.

Reply:
@briangustafson Any suggestion on how to filter compute shapes ? e.g. I only wanted to allow *VM compute shapes* setup on my instances under the nested-kvm sample.